packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx

```diff
@@ -122,10 +120,10 @@ const TranslationInput = (props: TranslationInputProps) => {
     if (onFocus) onFocus();
   }, [props.isCollapsed, onFocus, onToggle]);
 
-  // This checks if the content in the textarea overflows the minimum
-  // amount of lines it should have when collapsed
+  // This checks if the content in the textarea is greater than 38 (one row). If it is, then the toggle button will be shown.
```

Comment:
I'm sorry but I'd like to ask about this so I can understand it better. What does 38 mean? Is it the number of characters that fill one row? Is it the height of the row? Is it another measurement?

Reply:
@CarlosCortizasCT sure, let me try to explain. If I understood everything correctly, [this](https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/localized-multiline-text-input/src/translation-input.tsx#L89) `handleHeightChange` is being referenced as a callback to the function `handleHeightChange` in [multiline-input.tsx](https://github.com/commercetools/ui-kit/blob/main/packages/components/inputs/input-utils/src/multiline-input/multiline-input.tsx#L40). 
Where we calculate the row count with dividing `scrollHeight` and `rowHeight` of the textarea. 
If it's only one row in the new theme the result will be number 38 and in the old theme it'll be 30.
It's somewhat complex written but when I console logged the `rowHeight` value I grasped it that it if we go above 38 it means it's not a single line anymore.
<img width="337" alt="Screenshot 2023-05-02 at 13 12 11" src="https://user-images.githubusercontent.com/52276952/235665415-b3db349d-8a7d-4f1f-922b-93d8dc2970ce.png">

Does it makes any more sense now? 😅

Reply:
Ok, thanks for the explanation.
So I understand 38 (in the new theme) in the height of the textarea in pixels as we can see in the browser inspector:
![image](https://user-images.githubusercontent.com/97907068/235697432-e395c6d6-e60f-4cc1-af0e-6e7a0cd5b141.png)

Looking at the numbers you shared, it matches the line-height of the component:
![image](https://user-images.githubusercontent.com/97907068/235697673-d80c4ffc-d958-40bb-96f0-fb261c6c7dca.png)

So at the end is seems like a complex way of calculating the height of the input.

The little drawback I see here is that we will have this 38 _magic number_ in our code which will work until we change the style of the components.
Perhaps we can calculate that number using the `line-height` and padding `styles` after its first render:
```js
const elementComputedStyles = getComputedStyle(<element_ref>);
const lineHeight = parseInt(elementComputedStyles.getPropertyValue('line-height'), 10); // -> 22
const paddingTop = parseInt(elementComputedStyles.getPropertyValue('padding-top'), 10); // -> 8
const paddingBottom = parseInt(elementComputedStyles.getPropertyValue('padding-bottom'), 10); // -> 8
const elementHeight = lineHeight + paddingTop + paddingBottom; // -> 38
```

What do you think?

Reply:
Yes, it absolutely makes more sense. Thank you for that! 🙇‍♀️ 
So I implemented it here - [b49e37c](https://github.com/commercetools/ui-kit/pull/2511/commits/b49e37c30737f23352f2636df740a067df423e2b). Let me know it there's something else.